js/components/utils.js

```diff
@@ -953,6 +953,27 @@ var Utils = {
     }
     return false;
   },
+
+  /**
+   * the function returns the maximum width of the elements
+   * @param {...Element} DOM Element
+   * @returns {number} - max width of the elements
+   */
+  getMaxElementWidth: function() {
+    var argumentsArray = Array.prototype.slice.call(arguments);
+    var startWidth = 0;
```

Comment:
Should we use maxWidth instead of startWidth, since the function is returning the max element width?

Reply:
it was removed.